sale_delivery_rate/demo/delivery_carrier.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2017 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+    <record id="demo_carrier_1" model="delivery.carrier">
```

Comment:
nice I like your idea of adding demo in front of the id. I think I'll start doing that too :)